platform-ui-skin/src/main/webapp/skin/less/social/skin/Activity/Style.less

```diff
@@ -1854,6 +1854,27 @@ blockquote:after {
 	}
 }
 
+.uiActivityStreamWrapper.uiWikiActivity {
+  .wikiViewChanges {
+     background: @greyColorLighten1Opacity2;
+     padding: 10px;
+  }
+}
+
+.uiActivityStreamWrapper.uiPollActivity {
+  .uiPollVoteAction {
```

Comment:
It's better IMO to apply a generic class in activities actions buttons like `uiActivityAction` and then define its style here.
This will avoid to having define the same style multiple times for each Activity action.

Reply:
Done